helm/idol-nifi/resources/nifi-import-flow.sh

```diff
@@ -172,7 +172,7 @@ if [ 0 != ${#NEW_PROCESS_GROUP_IDS[@]} ]; then
     SVCSTART_PROCESS_GROUP_IDS=("${NEW_PROCESS_GROUP_IDS[@]}")
     START_PROCESS_GROUP_IDS=("${NEW_PROCESS_GROUP_IDS[@]}")
 
-    for i in {1..12}
+    for ((i=1; i <= ${IDOL_NIFI_SERVICE_START_RETRIES:-12}; i++))
```

Comment:
Not sure why it was 12 before, but it does seem like a high value. Does it need to be this high by default?

Reply:
It's been 12 from the start (something like 5s sleep = 60s of retry) - reduced to 3 in [c13fe54](https://github.com/opentext-idol/idol-containers-toolkit/pull/97/commits/c13fe548aea3f29129ff819730e9f5864cbf8b84)